com.woltlab.wcf/templates/trophyList.tpl

```diff
@@ -3,7 +3,7 @@
 {capture assign='contentHeader'}
 	<header class="contentHeader messageGroupContentHeader">
 		<div class="contentHeaderTitle">
-			<h1 class="contentTitle">{$category->getTitle()}</h1>
+			<h1 class="contentTitle">{$category->getTitle()} <span class="badge badgeInverse">{#$items}</span></h1>
```

Comment:
Neither do we always show it for zero values, nor do we always hide it. It depends on the context, in particular if the `0` has an actual meaning.

For example, the draft section in conversations do display a zero to indicate that there are no drafts, avoiding confusing with numbers only seen when there are unread items. But in case of the trophy list (and others) there is already a clear indication, that there are absolutely no items, therefore displaying the zero has no purpose. Also you're already on that very page, it is obvious that there are no items.